gcslock.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
```

Comment:
True but it doesn't hurt to have it and leaving that line intact seems more tolerant because it handles people execing it or sourcing it.